src/index.js

```diff
@@ -1225,6 +1225,23 @@ class CloudManagerAPI {
       throw e
     })
   }
+
+  /**
+   * Make a Post to Commerce API
+   *
+   * @param {string} programId - the program id
+   * @param {string} environmentId - the environment id
+   * @param {object} requestBody - the request body
+   * @returns {Promise<object>} a truthy value
+   */
+  postCommerceCLICommand (programId, environmentId, requestBody) {
+    const resourcePath = `/program/${programId}/environment/${environmentId}/runtime/commerce/cli/`
```

Comment:
@rwalter215 this path should be derived from a HAL link as explained in https://github.com/adobe/aio-lib-cloudmanager/blob/main/.github/CONTRIBUTING.md#coding-patterns-and-styles

Reply:
As of now we cannot get the path from a HAL link, that functionality has not been added to the api yet. Can I keep this code the way it is, or is this there a different approach you'd prefer.

Reply:
I can't think of another alternative.

Reply:
I implemented retrieving the url from the HAL link, we'll just deal with this functionality not being available until the api supports it.